cmd/sql/flow.go

```diff
@@ -272,13 +280,16 @@ func generateCommand() *cobra.Command {
 		SilenceUsage: true,
 	}
 	cmd.SetHelpFunc(executeHelp)
-	cmd.Flags().BoolVar(&generateTasks, "generate-tasks", false, "")
+	cmd.Flags().BoolVar(&generateTasks, "generate-tasks", true, "")
+	cmd.Flags().BoolVar(&noGenerateTasks, "no-generate-tasks", false, "")
```

Comment:
Because, we want to reflect what we expose in the typer cli. In typer cli a boolean `--generate-task` flag automatically receives a `--no-...` option, but cobra https://github.com/spf13/pflag/issues/214 does not. :(

Reply:
Yes, that's true, we could only use one flag, but we might want to change this to have `--generate-tasks/--render-tasks`. WDYT? @pankajkoti @tatiana @dimberman? Should we have only option `--generate-tasks` or `--render-tasks`? I think this makes sense once we know for sure what the default long-term will be.

Reply:
For now, I would suggest we keep it as it is now to make it fully compatible with the current interface in sql-cli.

In the future, we might want to release a change for this.

Reply:
In my opinion, at the moment, it makes sense to have the two flags `--generate-tasks` and `--no-generate-tasks` as that is what the help command output looks like with the current version of sql-cli (flow).